flocker/control/test/persisted_model.json

```diff
@@ -34,6 +34,7 @@
                     "running": [
                         "__builtin__.bool"
                     ],
+                    "swappiness": [],
```

Comment:
:question: Oh, I'd forgotten about this model change test. I'll have to remind myself how this works....